src/relayserver/RegistrationManager.ts

```diff
@@ -166,7 +162,7 @@ export class RegistrationManager {
     const registerEvents = relayRegisteredEvents.filter(
       (eventData: EventData) =>
         isRegistrationValid(eventData, this.config, this.managerAddress))
-    return reduceToLatestTx(registerEvents)
+    return getLatestEventData(registerEvents)
```

Comment:
I don't think this does what you think it does: if my config matches a previous RelayRegistered  (e.g. I changed it, caused the relay to re-register and now changed it back) then the above filter on isRegistrationValid will succeed - for some previous configuration. you must first do getLatest, and only then check if it is valid.

Reply:
You are absolutely right... In fact, I do not need to filter at all. Fixing,